nexus_constructor/geometry/disk_chopper/disk_chopper_geometry_creator.py

```diff
@@ -0,0 +1,394 @@
+from typing import List, Tuple
+
+import numpy as np
+from PySide2.QtGui import QVector3D
+
+from nexus_constructor.geometry import OFFGeometryNoNexus
+from nexus_constructor.geometry.disk_chopper.chopper_details import ChopperDetails
+
+ARROW_SIZE = 0.1
```

Comment:
It would be good to have a comment for each of these constants to explain what they are used for.